.github/CODEOWNERS

```diff
@@ -1158,9 +1158,11 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 ### Observability Plugins
 
 # Observability AI Assistant
-x-pack/test/observability_ai_assistant_api_integration @elastic/obs-ai-assistant
-x-pack/test/observability_ai_assistant_functional @elastic/obs-ai-assistant
-x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai-assistant
+/x-pack/test_serverless/api_integration/test_suites/common/data_usage @elastic/obs-ai-assistant
+/x-pack/test_serverless/functional/test_suites/common/data_usage @elastic/obs-ai-assistant
```

Comment:
I don't think these should be owned by the Obs AI Assistant team. 
@neptunian WDYT?

Reply:
We've been assigning all the `data_usage` related code to `@elastic/obs-ai-assistant` and `@elastic/security` as there isn't a data usage team and we haven't handed it off to any team yet.  I think we should probably continue with that until there is a handoff, likely to `@elastic/kibana-management`.

Reply:
https://github.com/elastic/kibana/pull/200147/commits/774d25e53332ee23d2c5abe802839a6e66c9f39b